source/accessories/dac/index.md

```diff
@@ -29,54 +27,54 @@ The AMDC DAC expansion board was designed with the following requirements:
 | 3          | No Connect  | No Connect |
 | 4          | No Connect  | No Connect |
 | 5          | No Connect  | No Connect |
 | 6          | No Connect  | No Connect |
-| 7          | No Connect  | `SPI_IP`   |
-| 8          | `STS_A`     | `SPI_IM`   |
-| 9          | `STS_B`     | No Connect |
-| 10         | `STS_C`     | No Connect |
+| 7          | No Connect  | No Connect |
+| 8          | `STS_A`     | No Connect |
+| 9          | `STS_B`     | `OUT3_P`   |
+| 10         | `STS_C`     | `OUT3_N`   |
 | 11         | No Connect  | `GND`      |
-| 12         | No Connect  | No Connect |
-| 13         | `+5V`       | No Connect |
-| 14         | No Connect  | No Connect |
-| 15         | `GND`       | No Connect |
+| 12         | No Connect  | `OUT1_P`   |
+| 13         | `+5V`       | `OUT1_N`   |
+| 14         | No Connect  | `OUT2_P`   |
+| 15         | `GND`       | `OUT2_N`   |
 
 ### Isolators
 
-Before the digital signals are routed to the jumpers, they are isolated from the AMDC through digital isolators. The power signals are also isolated within the DC-to-DC converters. The status lines from the Power Stack Inverter are isolated using a single [ADUM130E1BRWZ](https://www.analog.com/media/en/technical-documentation/data-sheets/ADuM130D_130E_131D_131E.pdf) IC. The IsoSPI signals are isolated by the IsoSPI transceiver and do not require additional isolation.
+Before the digital signals are routed to the jumpers, they are isolated from the AMDC through digital isolators. The power signals are also isolated within the DC-to-DC converters. The status lines from the Power Stack Inverter and GPIO Signal-Differentiator are each isolated using their own [ADUM130E1BRWZ](https://www.analog.com/media/en/technical-documentation/data-sheets/ADuM130D_130E_131D_131E.pdf) IC.
 
-### Jumper Blocks
+### Jumper Blocks/DIP Switch
 
-There is a total of eight jumpers on the DAC board that configure the expected pinout from the DB-15 connector. 
+There are a total of eight jumpers on the DAC board, along with a single 6-channel DIP switch, that configure the expected pinout from the DB-15 connector. 
 
 - Jumpers JP1, JP2, and JP3 route the SPI signals used to communicate with the DAC 
 - Jumpers JP4 and JP5 route the power signals to the board. 
-- Jumpers JP6, JP7, and JP8 enable or disable the SPI signal digital isolator. 
+- Jumpers JP6, JP7, and JP8 enable or disable the SPI signal digital isolator in the Power Stack Configuration. 
 
 All eight jumpers are critical to the operation of the DAC board and must be installed on the appropriate side of the jumper blocks as designated by the silkscreen. An example installation of the Power Stack Inverter configuration can be seen below:
 
-```{image} images/amdc-dac-jmp1.jpg
-:width: 20em
-```
+<img src="images/amdc-dac-jmp1.jpg" style="max-width:580px">
 
-```{image} images/amdc-dac-jmp2.jpg
-:width: 20em
-```
+<br>
+<br>
+
+<img src="images/amdc-dac-jmp2.jpg" style="max-width:580px">
```

Comment:
@codecubepi these images do not render when the website it built using Sphinx. You need to use the previous image include format, not the HTML "<img>" tag

Reply:
@npetersen2 Huh. To my knowledge I never updated those images so I'm not sure why that would have been changed. Either way I reverted it. Try again now?